subject_files_status_for_dpdash.py

```diff
@@ -34,6 +35,27 @@ def _latest_mtime(p: Path) -> str:
     return datetime.fromtimestamp(latest).strftime('%Y-%m-%d')
 
 
+def check_upenn_cnb(subject_raw_path: Path) -> bool:
+    '''Check if there are SPLLT-A and NOSPLLT in the Penn CNB json file'''
+    json_paths = list((subject_raw_path / 'surveys').glob('*UPENN.json'))
+
+    if len(json_paths) == 0:  # No file
+        return False
+
+    with open(json_paths[0], 'r') as fp:
+        data = json.load(fp)
+
+    if len(data) != 2:
+        return False
+
+    session_batteries = [x['session_battery'] for x in data]
+    spllta_check = 'SPLLT-A' in session_batteries
+    nospllt_check = ('ProNET_NOSPLLT' in session_batteries or
+                     'PRESCIENT_NOSPLLT_C1' in session_batteries)
+
+    return spllta_check & nospllt_check
```

Comment:
Why is it not `and`? I see you used `&`.

Reply:
`&` and `and` works the same. Replaced to `*` to return `int` in 3ca36d3. Thanks.